conditional/util/email.py

```diff
@@ -0,0 +1,51 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.utils import formatdate
+from email.mime.multipart import MIMEMultipart
+from datetime import date
+from conditional import app
+
+def send_absent_hm_attendance_emails(absent_members):
+    today_date = date.today().strftime("%m/%d")
+    subject = "You were marked as absent for the {} House Meeting.".format(today_date)
+    body = subject + "\n\nIf this is a mistake please message the evaluations director and let them know."
+    # doing this uses the same connection for all emails sent to absent members
+    send_email(absent_members, subject, body)
+
+def send_present_hm_attendance_email(member):
+    today_date = date.today().strftime("%m/%d")
+    subject = "You were marked as present for the {} House Meeting.".format(today_date)
+    body = subject
+    # doing this uses the same connection for all emails sent to absent members
+    send_email([member], subject, body)
+
+def send_attenance_requirement_met_email(user):
+    subject = "You have attended 30 Directorship Meetings"
```

Comment:
Is this number defined somewhere that you could pull from, rather than it being a magic number here? Also what about people in a coop group that only need 15?